### PR TITLE
deploy to PyPI using travis

### PR DESCRIPTION
**Pull request by osherdp**
The deploying process requires two steps:
* Update the ```__version__``` variable on ```setup.py```
* Make a new release (and automatically a new git tag) on Github
Then, a new build will be created for the new tag, and after test stages for each Python version pass, there is another stage for deploying

Two changes should be made to my code, by anyone with access rights to the PyPI package:
* Changing the username
* Setting (and decrypting) the password: Install ruby, then follow the short steps on ```.travis.yml```

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -8,6 +12,22 @@ python:
...
+    install: skip
+    before_script: skip
+    script: skip
+    deploy:
+      stage: deploy
+      provider: pypi
+      on:
+        tags: true
+      distributions: sdist bdist_wheel
+      user: <PyPI user>
```
Comment (krasoffski) [inline .travis.yml]: There is  a such pypi user `reportportal`, so I would set `user: reportportal` (https://pypi.org/project/reportportal-client/) for now and @DzmitryHumianiuk  provided encrypted password for this user.

**Comment by osherdp** [comment]
Retry Deepcode

**Comment by ch-t** [comment]
Retry Deepcode

**Comment by krasoffski** [comment]
@DzmitryHumianiuk  
This PR contains almost all steps required for making PyPI release based on GitHub release atomatically. This means that when you add user/password of PyPI acount to travis system, releases will be created automatically as soon as you create release in the GitHub.
Owner of travis job has to configure access to PyPI reportportal account (this described in the documentation of travis CI and mentioned above.

**Pull request merged**